typed-racket-lib/typed-racket/types/base-abbrev.rkt

```diff
@@ -197,6 +197,12 @@
 (define (simple-> doms rng)
   (->* doms rng))
 
+(define (simple-arrow doms rng)
+  (-Arrow doms rng))
+
+(define (simple->values doms rng)
+  (->* doms (make-Values (map -result rng))))
```

Comment:
I read this identifier as "simple to values"... but its really more like "simple arrow with values" or something, right?

Reply:
Yes. This just exists so that the expansion of type serialization is smaller.